test/test.js

```diff
@@ -1,8 +1,195 @@
+		"outer.inner.key changes the observation"
+	);
+});
+
+QUnit.test("to method works", function() {
+	var outer = {inner: {key: "hello"}};
+	var setProp = canValue.to(outer, "inner.key");
+
+	// Getting the value shouldn’t work
+	QUnit.equal(canReflect.getValue(setProp), undefined, "getting doesn’t work");
+
+	// Test setting the value
+	canReflect.setValue(setProp, "aloha");
+	QUnit.equal(outer.inner.key, "aloha", "setting works");
+});
+
+QUnit.test("to method returns an observable with a helpful name", function() {
+	var outer = {inner: {key: "hello"}};
+	var observable = canValue.to(outer, "inner.key");
+	QUnit.equal(
+		canReflect.getName(observable),
+		"CanValueToObservable<Object{}.inner.key>",
+		"observable has the correct name"
+	);
+});
+
+QUnit.test("to method observable has dependency data", function(assert) {
+	var outer = new SimpleMap({inner: new SimpleMap({key: "hello"})});
+	var observable = canValue.to(outer, "inner.key");
+
+	// Check outer.inner’s dependency data
+	var innerDepData = canReflectDeps.getDependencyDataOf(outer, "inner");
+	assert.notOk(innerDepData.whatIChange, "outer.inner does not change anything");
+	assert.notOk(innerDepData.whatChangesMe.derive, "outer.inner does not have derive dependencies");
+	assert.equal(
+		innerDepData.whatChangesMe.mutate.keyDependencies.size,
+		0,
+		"outer.inner has zero mutate.keyDependencies"
+	);
+	assert.equal(
+		innerDepData.whatChangesMe.mutate.valueDependencies.size,
+		1,
+		"outer.inner has one mutate.valueDependencies"
+	);
+	assert.ok(
+		innerDepData.whatChangesMe.mutate.valueDependencies.has(observable),
+		"outer.inner is changed by observable"
+	);
+
+	// Check outer.inner.key’s dependency data
+	var keyDepData = canReflectDeps.getDependencyDataOf(outer.get("inner"), "key");
+	assert.notOk(keyDepData.whatIChange, "outer.inner.key does not change anything");
+	assert.notOk(keyDepData.whatChangesMe.derive, "outer.inner.key does not have derive dependencies");
+	assert.equal(
+		keyDepData.whatChangesMe.mutate.keyDependencies.size,
+		0,
+		"outer.inner.key has zero mutate.keyDependencies"
+	);
+	assert.equal(
+		keyDepData.whatChangesMe.mutate.valueDependencies.size,
+		1,
+		"outer.inner.key has one mutate.valueDependencies"
+	);
+	assert.ok(
+		keyDepData.whatChangesMe.mutate.valueDependencies.has(observable),
+		"outer.inner.key is changed by observable"
+	);
+
+	// Check observable’s dependency data
+	var observableDepData = canReflectDeps.getDependencyDataOf(observable);
+	assert.notOk(observableDepData.whatChangesMe, "observable is not changed by anything");
+	assert.notOk(observableDepData.whatIChange.derive, "observable does not have derive dependencies");
+	assert.notOk(
+		observableDepData.whatIChange.mutate.valueDependencies,
+		"observable has zero mutate.valueDependencies"
+	);
+	assert.equal(
+		observableDepData.whatIChange.mutate.keyDependencies.size,
+		1,
+		"observable has one mutate.keyDependencies"
```

Comment:
Is this correct, that in the case of `observable` -> `outer.inner.key`, the observable should only register that it modifies `outer.inner`, and not `outer` as well?